lib/snapshot/collector.rb

```diff
@@ -49,8 +50,10 @@ def self.attachments(containing)
               (subtest2["Subtests"] || []).each do |subtest3|
                 (subtest3["ActivitySummaries"] || []).each do |activity|
                   # We now check if it's the rotation gesture, because that's the only thing we care about
-                  was_snapshot = activity["Title"] == "Set device orientation to Unknown"
-                  activities << activity if was_snapshot
+                  if activity["Title"] == "Set device orientation to Unknown"
+                    to_store << activity["Attachments"].last["FileName"]
+                    count += 1
```

Comment:
Yes, did not see that. Changing it right now.
On Sun, Dec 20, 2015 at 14:01 Felix Krause <notifications@github.com> wrote:

> In lib/snapshot/collector.rb
> <https://github.com/fastlane/snapshot/pull/359#discussion_r48100126>:
>
> > @@ -49,8 +50,10 @@ def self.attachments(containing)
> >                (subtest2["Subtests"] || []).each do |subtest3|
> >                  (subtest3["ActivitySummaries"] || []).each do |activity|
> >                    # We now check if it's the rotation gesture, because that's the only thing we care about
> > -                  was_snapshot = activity["Title"] == "Set device orientation to Unknown"
> > -                  activities << activity if was_snapshot
> > +                  if activity["Title"] == "Set device orientation to Unknown"
> > +                    to_store << activity["Attachments"].last["FileName"]
> > +                    count += 1
>
> We don't need a count, you could easily just use to_store.count, right?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/fastlane/snapshot/pull/359/files#r48100126>.
>